fuel_utility.py

```diff
@@ -20,89 +20,114 @@
 from docopt import docopt
 import zipfile
 import io
-import xml.etree.ElementTree as ET
-import xml.dom.minidom
+from lxml import etree as ET
 import os.path
 
-FUEL_URI="https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models"
-WAIT_BETWEEN_DOWNLOADS_IN_SECONDS=5
+FUEL_URI = 'https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models'
+WAIT_BETWEEN_DOWNLOADS_IN_SECONDS = 5
```

Comment:
Why is this wait required?

Reply:
Thought it would be prudent to put in a mechanism to throttle the downloads - so the team at Fuel doesn’t have to worry about big spikes in traffic if this script is used broadly. 5 seconds between download seemed like a manageable download rate.